### ISSUE TITLE
Переместить образы на свой собственный сервер

### ISSUE THREAD

**Issue by BlitDev** [Ubuntu-Sway/Ubuntu-Sway-Remix#11]
Можно купить VDS'ку, распространять там образы в виде сжатых **zstd** архивов, должны весить не много.
Можно даже не делать свой интерфейс как на других сайтах, а просто запилить HTTP сервер с деревом каталогов.
Загрузка файлов с GitHub очень медленная + в случае блокировки GitHub (хотя он вроде бы опроверг это), дистрибутив останется.
Конечно в идеале запилить свой сайт с нуля, возможно я даже займусь этим.

**Comment by Sunderland93** [comment]
SourceForge, как вариант?

**Comment by amikha1lov** [comment]
> SourceForge, как вариант?

Привет. Можем помочь с дизайном сайта и самим сайтом.

**Comment by BlitDev** [comment]
> SourceForge, как вариант?

Можно, но лучше свой сайт :)

**Comment by amikha1lov** [comment]
> Сайт начальный есть https://github.com/Ubuntu-Sway/Ubuntu-Sway.github.io в качестве сервера загрузок можно на время Source Forge прикрутить. У меня на данный момент нет ресурсов на содержание своего сайта

Я и @Sabadil готовы оплачивать  VPS/VDS на бегете https://beget.com/ru/vps

**Comment by BlitDev** [comment]
> > Сайт начальный есть https://github.com/Ubuntu-Sway/Ubuntu-Sway.github.io в качестве сервера загрузок можно на время Source Forge прикрутить. У меня на данный момент нет ресурсов на содержание своего сайта
> 
> Я и @Sabadil готовы оплачивать VPS/VDS на бегете https://beget.com/ru/vps

Если будет задача на C# или другой известной мне технологии, могу помочь с разработкой :)

**Comment by Sunderland93** [comment]
@amikha1lov звучит здорово! Сайтом на данный момент занимается @iwalfy , думаю нужно будет заранее продумать всю структуру

**Comment by amikha1lov** [comment]
> @amikha1lov звучит здорово! Сайтом на данный момент занимается @iwalfy , думаю нужно будет заранее продумать всю структуру

он есть в телеге или матриксе?

**Comment by Sunderland93** [comment]
В телеге есть, @lebedinetss насчет матрикса не знаю

**Comment by amikha1lov** [comment]
> @lebedinetss

написал тебе в телеге по поводу сервака и домена

**Comment by Sunderland93** [comment]
Сервер создал, осталось приобрести домен. Самый подходящий - .com, на beget стоит 990 в год

**Comment by Sunderland93** [comment]
Все готово к переносу сайта

**Comment by amikha1lov** [comment]
@Sunderland93 думаю можно закрывать

**Comment by BlitDev** [comment]
> 

Да, @Sunderland93 Сделайте коммит с новыми ссылками, укажите в начале ссылочку на это issue #11, и потом я его закрою.
